src/main/java/org/folio/repository/holdings/LoadStatus.java

```diff
@@ -4,7 +4,7 @@
 
 public enum LoadStatus {
 
-  COMPLETED("Completed"), IN_PROGRESS("In progress"), FAILED("Failed");
+  COMPLETED("Completed"), IN_PROGRESS("In progress"), FAILED("Failed"), NONE("None");
```

Comment:
If we get a status - NONE, then we should make a POST request to RM API for staging data. Are we handling that?

Reply:
Yes, we make POST request when we get any status except "In progress"

Reply:
👍